scss/components/side-nav.scss

```diff
@@ -74,12 +74,34 @@ $block: #{$fd-namespace}-side-nav;
 
     &__link,
     &__sublink {
+        display: flex;
+        align-items: flex-start;
         padding: $fd-side-nav-link-padding;
         color: $fd-side-nav-link-color;
-        display: block;
         @include fd-type("0", "normal");
 
+        .fd-side-nav__link-icon {
+            line-height: 20px;
+
+            &--s {
+                line-height: 19px;
+            }
+
+            &--m {
+                line-height: 20px;
+            }
+
+            &--l {
+                line-height: 24px;
+            }
+
+            &--xl {
+                line-height: 28px;
+            }
+        }
+
```

Comment:
_Seems_ excessive to make another class just for this.  Can't we leverage the existing classes that already identify icons and/or the arrow to expand/collapse children links?